chart/compass/values.yaml

```diff
@@ -914,7 +914,7 @@ global:
     appTemplatesProductLabel: "systemRole"
     systemSourceKey: "prop"
     appTemplates: []
-    templatePlaceholderToSystemKeyMappings: '[ { "placeholder_name": "name", "system_key": "displayName" }, { "placeholder_name": "display-name", "system_key": "displayName" }, { "placeholder_name": "systemNumber", "system_key": "systemNumber" }, { "placeholder_name": "productId", "system_key": "productId" }, { "placeholder_name": "ppmsProductVersionId", "system_key": "ppmsProductVersionId", "optional": true }, { "placeholder_name": "region", "system_key": "additionalAttributes.systemSCPLandscapeID", "optional": true }, { "placeholder_name": "description", "system_key": "productDescription", "optional": true }, { "placeholder_name": "baseUrl", "system_key": "additionalUrls.mainUrl", "optional": true }, { "placeholder_name": "providerName", "system_key": "infrastructureProvider", "optional": true } ]'
+    templatePlaceholderToSystemKeyMappings: '[ { "placeholder_name": "name", "system_key": "$.displayName" }, { "placeholder_name": "display-name", "system_key": "$.displayName" }, { "placeholder_name": "systemNumber", "system_key": "$.systemNumber" }, { "placeholder_name": "productId", "system_key": "$.productId" }, { "placeholder_name": "tenant-id", "system_key": "$.systemNumber" }, { "placeholder_name": "ppmsProductVersionId", "system_key": "$.ppmsProductVersionId", "optional": true }, { "placeholder_name": "region", "system_key": "$.additionalAttributes.systemSCPLandscapeID", "optional": true }, { "placeholder_name": "description", "system_key": "$.productDescription", "optional": true }, { "placeholder_name": "baseUrl", "system_key": "$.additionalUrls.mainUrl", "optional": true }, { "placeholder_name": "providerName", "system_key": "$.infrastructureProvider", "optional": true } ]'
```

Comment:
How much effort would it be to not define the `tenant-id` placeholder here and rather define in the internal config with `jsonPath`? Reason being is that having it here indicates that in general `$.systemNumber` is a reasonable default for the local tenant id, which is not the case. There isn't a well-established value which System Fetcher can rely on being "in most cases" the right local tenant id. So putting it here is very S4-biased.

Also this makes the S4 app template (in the internal config) very coupled wit this `templatePlaceholderToSystemKeyMappings`, so at least from a readability and troubleshooting standpoint it would be better to define the placeholder with it's jsonPath in the internal config as part of the S4 app template.

At the end of the day this is not something that's a MUST fix, but I think it would overall result in a better setup/configuration. So pls if you agree, just consider what's the effort doing this and if it would risk the timely delivery of this item, I'd postpone this tweak for a later item/task.